xml/telnet_banners.xml

```diff
@@ -816,4 +816,680 @@
+  <fingerprint pattern="Solaris (.*)">
+    <description>Solaris</description>
+    <!-- Seattle Community Network Sun Solaris 1.1.1.B\nPlease login as 'visitor' if you are a visitorn\n\nSunOS UNIX (scn)\n\n\nlogin:-->
+    <example _encoding="base64" os.version="1.1.1.B">
+      U2VhdHRsZSBDb21tdW5pdHkgTmV0d29yayBTdW4gU29sYXJpcyAxLjEuMS5CClBsZWFzZSBsb2dpbiBhcyAndml
+      zaXRvcicgaWYgeW91IGFyZSBhIHZpc2l0b3IKCgpTdW5PUyBVTklYIChzY24pCgoKCmxvZ2luOg==
+    </example>
+    <param pos="0" name="hw.vendor" value="Sun"/>
+    <param pos="0" name="hw.family" value="Solaris"/>
+    <param pos="0" name="hw.product" value="Solaris"/>
+    <param pos="1" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="Digital UNIX \(([^)]+).*">
+    <description>Digital Unix</description>
+    <!-- Digital UNIX (journal) (ttyp2)\n\n\nlogin: -->
+    <example _encoding="base64" host.name="journal">
+      RGlnaXRhbCBVTklYIChqb3VybmFsKSAodHR5cDIpCgoKCmxvZ2luOiA=
+    </example>
+    <param pos="0" name="hw.vendor" value="HP"/>
+    <param pos="0" name="hw.family" value="Digital Unix"/>
+    <param pos="0" name="hw.product" value="Digital Unix"/>
+    <param pos="1" name="host.name"/>`
+  </fingerprint>
+  <fingerprint pattern="^(?m)Compaq Tru64 UNIX V(.*) \(Rev. (.*\d)\) .*">
+    <description>Compaq Tru64 UNIX V</description>
+    <!-- Compaq Tru64 UNIX V5.1B (Rev. 2650) (docalpha) (pts/11)\n\n\n\n\nlogin: -->
+    <example _encoding="base64" os.version="5.1B" os.rev="2650">
+     Q29tcGFxIFRydTY0IFVOSVggVjUuMUIgKFJldi4gMjY1MCkgKGRvY2FscGhhKSAocHRzLzExKQoKCgoKCmxvZ2luOg== 
+    </example>
+    <param pos="0" name="hw.vendor" value="HP"/>
+    <param pos="0" name="hw.family" value="Digital Unix"/>
+    <param pos="0" name="hw.product" value="TRU64"/>
+    <param pos="1" name="os.version"/>
+    <param pos="2" name="os.rev"/>
+  </fingerprint>
+  <fingerprint pattern="HP-UX ([^ ]+) [A-Z]\.([^ ]+) ([^ ]+) ([^ ]+)\s([^ ]+\)).*$">
+    <description>System HP-UX</description>
+    <!-- HP-UX ctout B.11.11 U 9000/800 (tc)\nlogin: -->
+    <example _encoding="base64" host.name="ctout" os.version="11.11" hw.series="9000/800" hw.model="(tc)">
+      SFAtVVggY3RvdXQgQi4xMS4xMSBVIDkwMDAvODAwICh0YykKCmxvZ2luOiA=
+    </example>
+    <param pos="0" name="hw.vendor" value="HP"/>
+    <param pos="0" name="hw.family" value="HP-UX"/>
+    <param pos="0" name="hw.product" value="HP-UX"/>
+    <param pos="1" name="host.name"/>
+    <param pos="2" name="os.version"/>
+    <param pos="3" name="hw.version"/>
+    <param pos="4" name="hw.series"/>
+    <param pos="5" name="hw.model"/>
+  </fingerprint>
+  <fingerprint pattern="Data ONTAP">
+    <description>A NetApp apliance</description>
+    <!-- Data ONTAP (s500.)\nlogin: -->
+    <example _encoding="base64">RGF0YSBPTlRBUCAoczUwMC4pCmxvZ2luOiA=</example>
+    <param pos="0" name="hw.vendor" value="NetApp"/>
+    <param pos="0" name="hw.family" value="Data ONTAP"/>
+    <param pos="0" name="hw.device" value="Storage"/>
```

Comment:
should this be `Server?

Reply:
No I do not believe as this is what we currently assert and when looking online it is described as a storage device

Reply:
We are classifying this as a `Server` device in authenticated fingerprinting, perhaps we want to revise that.